Core/scripts/dirac-install.py

```diff
@@ -843,11 +843,17 @@ def downloadAndExtractTarball( tarsURL, pkgName, pkgVer, checkHash = True, cache
   tarName = "%s-%s.tar.gz" % ( pkgName, pkgVer )
   tarPath = os.path.join( cliParams.targetPath, tarName )
   tarFileURL = "%s/%s" % ( tarsURL, tarName )
+  tarFileCVMFS = "/cvmfs/dirac.egi.eu/installSource/%s" % tarName
```

Comment:
1.   No, it is not. We have just got our (DIRAC) area in the CVMFS repository in the egi.eu domain.
On some sites this will become automatically available. On some sites we will have to ask to enable it.
However, the installation does not depend on it. If it is not available, it will proceed as before by
the http download.
2. By your second question you mean your own cvmfs repository where you store your DIRAC extension software ? I think this possibility should be described in the repository description of your project. This needs some development but can be done.